frontend/device/android/device.lua

```diff
@@ -90,6 +91,15 @@ function Device:initNetworkManager(NetworkMgr)
     end
 end
 
+function Device:retrieveNetworkInfo()
+    local ssid, ip, gw = string.match(android.getNetworkInfo(), "(.*);(.*);(.*)")
```

Comment:
Is there a reason not to put this in luajit-launcher?

Reply:
in assets/android?. I don't know, it seems easy to expand here. Plus I'm not to excited messing with JNI.

Reply:
I'm not talking about JNI (that would be, e.g., making an array into a Lua table) but about not taking this backend logic to front.

Reply:
Basically I think that most forward-thinking, front here should just receive a table, grab android.getNetworkInfo().ssid, ip, and gw.

But since it's unlikely that we want more, it could also be okay to just return those three things. But it doesn't seem the best design to put that logic here.